source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,127 @@
+.. _c-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:atlas:`Atlas Search </atlas-search>` enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. Atlas Search indexes specify the behavior of
+the search and which fields to index.
+
+The following sections provide code examples that demonstrate how to create, list, update,
+and delete Atlas Search indexes.
+
+.. _c-atlas-search-index-create:
+
+Create a Search Index
+---------------------
+
+You can use the ``mongoc_collection_command_simple()`` function to create one or more
+Atlas Search indexes by using the ``createSearchIndexes`` command.
```

Comment:
S: to avoid the use/using repetition:

```suggestion
You can pass the ``createSearchIndexes`` command to the ``mongoc_collection_command_simple()`` function to create one or more
Atlas Search indexes.
```